src/Console/Deployer.php

```diff
@@ -241,4 +247,25 @@ private function copyProjectToOutputDirectory() : void
         $directoryMirror = new DirectoryMirror($this->fs);
         $directoryMirror->mirror($source, $target);
     }
+
+    private function injectPhpConfig(string $targetFile, array $configs, array $extensions = [])
+    {
+        array_walk($configs, function(&$value, $key) {
+            $value = $key . '=' .$value;
+        });
+
+        $extensions = array_map(function ($extension) {
+            if (!$this->fs->exists('.bref/output/.bref/bin/ext/' . $extension . '.so')) {
+                throw new \Exception('PHP Extension "'. $extension . '" does not exist');
+            }
+
+            return 'extension=' . $extension . '.so';
+        }, $extensions);
```

Comment:
> I couldn't find any library which can handle the several usages of the same key in an INI file:

Is that supposed to be done with sections? E.g. here is how it's done in my config files on my system:

```ini
[opcache]
zend_extension=/usr/local/opt/php/lib/php/20170718/opcache.so
```

Maybe we should put each extension with a section before? (the section could have the same name of the extension)

Also have you tried if it works with both redis and mongodb enabled?

As for PHP configuration this is different: https://stackoverflow.com/a/4651918/245552 The last value is taken, so that's fine for us because that means users can override values in the default `php.ini`.